### PR TITLE
Normal distributions

### PR DESCRIPTION
**Pull request by alebrigant**
Compute pdf associated to a point in the manifold of normal distributions.

### CODE REVIEW

**Comment by alebrigant** [comment]
I would like to get started on a jupyter notebook on information geometry, but for some reason I cannot 'git add' notebooks. I get the following error:
```
The following paths are ignored by one of your .gitignore files:
notebooks/06_information_geometry.ipynb
Use -f if you really want to add them.
```
Is this deliberate or is this a problem on my end ?

**Comment by ninamiolane** [comment]
You should be able to add notebooks with our settings @alebrigant 

Can you maybe check your `.gitignore` file? If there is the line `notebooks/06_information_geometry.ipynb` written there, it can be removed (we don't have it in the geomstats' master's .gitignore).

File: geomstats/geometry/beta_distributions.py [hunk 1]
```diff
@@ -104,6 +104,47 @@ def sample(self, point, n_samples=1):
...
+        Returns
+        -------
+        pdf : function
+            Probability density function of the beta distribution with
+            parameters provided by point.
+        """
+        geomstats.errors.check_belongs(point, self)
+        point = gs.to_ndarray(point, to_ndim=2)
+        a_params = point[:, 0]
+        b_params = point[:, 1]
```
Comment (nguigs) [inline geomstats/geometry/beta_distributions.py:127]: The ellipsis notation would work here and avoid using to_ndarray:
```python
a_params = point[..., 0]
b_params = point[..., 1]
```

File: geomstats/geometry/beta_distributions.py [hunk 2]
```diff
@@ -104,6 +104,47 @@ def sample(self, point, n_samples=1):
...
+            x : array-like, shape=[n_points,]
+                Points at which to compute the probability density function.
+            """
+            x = gs.array(x, gs.float32)
+            x = gs.to_ndarray(x, to_ndim=1)
+
+            pdf_at_x = [
+                gs.array(beta.pdf(x, a=a, b=b)) for a, b
+                in zip(a_params, b_params)]
+            pdf_at_x = gs.stack(pdf_at_x, axis=1)
```
Comment (nguigs) [inline geomstats/geometry/beta_distributions.py:143]: Using `axis=-1` works if there is only a single x or if there are many, this avoids using to_ndarray above

File: geomstats/geometry/normal_distributions.py [hunk 3]
```diff
@@ -71,6 +71,47 @@ def sample(self, point, n_samples=1):
...
+            x : array-like, shape=[n_points,]
+                Points at which to compute the probability density function.
+            """
+            x = gs.array(x, gs.float32)
+            x = gs.to_ndarray(x, to_ndim=1)
+
+            pdf_at_x = [
+                gs.array(norm.pdf(x, loc=mean, scale=std)) for mean, std
+                in zip(means, stds)]
+            pdf_at_x = gs.stack(pdf_at_x, axis=1)
```
Comment (nguigs) [inline geomstats/geometry/normal_distributions.py:110]: ditto

File: geomstats/geometry/normal_distributions.py [hunk 4]
```diff
@@ -71,6 +71,47 @@ def sample(self, point, n_samples=1):
...
+        Returns
+        -------
+        pdf : function
+            Probability density function of the normal distribution with
+            parameters provided by point.
+        """
+        geomstats.errors.check_belongs(point, self)
+        point = gs.to_ndarray(point, to_ndim=2)
+        means = point[:, 0]
+        stds = point[:, 1]
```
Comment (nguigs) [inline geomstats/geometry/normal_distributions.py:94]: ditto

**Review by nguigs — COMMENTED** [review]
Nice! Just a comment on handling vectorization.

**Comment by alebrigant** [comment]
Thanks for the comments @nguigs !

**Pull request merged**